.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,51 @@
+name: Test noir-wasm-testing
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          repository: ${{ github.repository }}
+
+      - name: Checkou
+        uses: actions/checkout@v3
+        with:
+          repository: aztecprotocol/noir-wasm-testing
+          path: noir-wasm-testing
```

Comment:
Why are these tests placed this in a separate repository? (in a separate organisation?). It would make more sense to include them in this repository imo.